e2e/tests/allergy/allergy-crud.e2e-spec.ts

```diff
@@ -0,0 +1,149 @@
+import { browser } from 'protractor';
+import { LoginPage } from '../../page-objects/login/login.po';
+import { MainPage } from '../../page-objects/main/main.po';
+import { AllergyDetailPage } from '../../page-objects/allergy/allergy-detail/allergy-dialog.po';
+import { NavigationService } from '../../services/navigation.service';
+import { TestUtil } from '../../utilities/test-util';
+import { FormService, FormData, ButtonState } from '../../services/form.service';
+
+declare const allure: any;
+
+describe('TC0002_AllergyManagement_e2e', () => {
+    const loginPage = new LoginPage();
+    const mainPage = new MainPage();
+    const allergyDetailPage = new AllergyDetailPage();
+
+    beforeAll(() => {
+	    NavigationService.navigateToHomePage(loginPage);
+	    NavigationService.login(loginPage);
+	    NavigationService.navigateToAllergyMaintenancePage(mainPage);
+    });
+
+    beforeEach(() => {
+        TestUtil.init('TC0002_AllergyManagement_e2e', 'Purpose: This TC is intended to verify the CRUD of an Allergy',
+            loginPage.appVersion, 'userName');
+    });
+
+    function getFormData(i?: number): FormData[]{
+        const baseAllergyValues = ['Name', 'A sign', 'A symptom'];
+        const empty = (i === undefined);
+        const formData: FormData[] = [{
+            field: allergyDetailPage.getNameInput().getElement(),
+            name: 'Name',
+            value: empty ? '' : 'Try #' + i + ': ' + baseAllergyValues[0]
+        }, {
+            field: allergyDetailPage.getSignsInput().getElement(),
+            name: 'Signs',
+            value: empty ? '' : 'Try #' + i + ': ' + baseAllergyValues[1]
+        }, {
+            field: allergyDetailPage.getSymptomsInput().getElement(),
+            name: 'Symptoms',
+            value: empty ? '' : 'Try #' + i + ': ' + baseAllergyValues[2]
+        }];
+        return (formData);
+    }
+
+    it('Open allergy creation dialog', () => {
```

Comment:
This step is documented twice. We only need to document the access to an screen once. Other accesses must use the navigation service to just navigate without additional documentation.

I've reviewed the formal report in Allure and the step "Access to the allergy screen" is the last one because we have re-used the same name for the TC TC0002_AllergyManagement_e2e. I think we could just remove the file "allergy-maintenance.e2e-spec.ts" or, change the TC name.